src/services/googleSheetsService.ts

```
@@ -0,0 +1,259 @@
+      await this.loadGoogleApi();
+      
+      // Esperar a que ambas APIs estén listas
+      const checkReady = setInterval(() => {
+        if (this.gapiInited && this.gisInited) {
+          clearInterval(checkReady);
+          this.isInitialized = true;
+          resolve();
+        }
+      }, 100);
+    });
+
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)